docs/src/tutorials/tutorial_6.md

```diff
@@ -0,0 +1,313 @@
+# **Tutorial 6:** Using Animations.jl to Create something with more Pep!
+
+In the last couple of tutorials you've learned the basics of Javis and some of the more advanced stuff like [`SubAction`](@ref).
+
+## Our goal
+
+The goal of this tutorial is to explain a new feature we have added in v0.2 of Javis. Before this every animation as basically linear.
```

Comment:
"every animation as basically linear." -> "was basically linear"

Reply:
Good catch. Should be fixed